### PR TITLE
Rel 620891 narrow the test set to reduce execution time

### PR DESCRIPTION
**Pull request by BartoszWierzbaRelativity**
Ignored tests:
- Copy tests
- Fileshare download tests
- File Fileshare upload tests

### CODE REVIEW

**Review by KamilKalitaRel — DISMISSED** [review]

**Review by mbartnic-relativity — APPROVED** [review]

File: test/DMLibTestCodeGen/DMLibDataTypeIdentifier.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+namespace DMLibTestCodeGen
+{
+    internal static class DMLibDataTypeIdentifier
+    {
+        public static bool IsLocal(DMLibDataType datatype)
+        {
+            return datatype == DMLibDataType.Local || datatype == DMLibDataType.Stream;
```
[the remark this answers is not in the document]
Comment (BartoszWierzbaRelativity) [reply test/DMLibTestCodeGen/DMLibDataTypeIdentifier.cs]: According to DMLibTestBase.IsLocal() yes.

File: test/DMLibTest/Cases/ResumeTest.cs [hunk 2]
```diff
@@ -67,7 +67,8 @@ public void MyTestCleanup()
         #endregion
 
         [TestCategory(Tag.Function)]
-        [DMLibTestMethodSet(DMLibTestMethodSet.AllValidDirection)]
+        [DMLibTestMethodSet(DMLibTestMethodSet.AllValidDirectionButNotCloudFileDest)]
+        [DMLibTestMethodSet(DMLibTestMethodSet.CloudFileDest, Ignore = true)]
```
[the remark this answers is not in the document]
Comment (BartoszWierzbaRelativity) [reply test/DMLibTest/Cases/ResumeTest.cs:71]: AllValidDirection was split into 2 attributes so CloudFileDest can be ignored.
AllValidDirection = AllValidDirectionButNotCloudFileDest + CloudFileDest

File: test/DMLibTest/Cases/LongFilePathTest.cs [hunk 3]
```diff
@@ -93,7 +93,8 @@ public void MyTestCleanup()
         #endregion
 
         [TestCategory(Tag.Function)]
-        [DMLibTestMethodSet(DMLibTestMethodSet.LocalSource)]
+        [DMLibTestMethodSet(DMLibTestMethodSet.LocalSourceButNotCloudFileDest)]
```
[the remark this answers is not in the document]
Comment (BartoszWierzbaRelativity) [reply test/DMLibTest/Cases/LongFilePathTest.cs:96]: Same thing. Isolate upload to file share and ignore it.

File: test/DMLibTest/Cases/BVT.cs [hunk 4]
```diff
@@ -103,7 +103,8 @@ public void TransferDirectoryDifferentSizeObject()
         }
 
         [TestCategory(Tag.BVT)]
-        [DMLibTestMethodSet(DMLibTestMethodSet.AllValidDirection)]
+        [DMLibTestMethodSet(DMLibTestMethodSet.AllValidDirectionButNotCloudFileDest)]
```
[the remark this answers is not in the document]
Comment (BartoszWierzbaRelativity) [reply test/DMLibTest/Cases/BVT.cs]: Sounds better. Done

**Review by mbartnic-relativity — APPROVED** [review]

**Review by KamilKalitaRel — DISMISSED** [review]

**Review by mbartnic-relativity — APPROVED** [review]

**Review by KamilKalitaRel — APPROVED** [review]

**Pull request merged**